wiki/modding/weeks/index.md

```diff
@@ -0,0 +1,69 @@
+---
+author: BerGP
+desc: How to make custom weeks
+title: Weeks
+lastUpdated: 2025-02-08T19:55:06.777Z
+---
+# Creating Weeks
+Weeks have to be created manually currently, sorry.
+
+## <h2 id="week-xml" sidebar="Week.xml">The XML</h2>
+
+Make the xml under ``./data/weeks/weeks/``. Now now, this whole "week xml" thing sounds complicated but trust me, it's not that hard.
+```xml
+<week name="PICO" chars="pico,bf,gf" sprite="week3">
+	<song>Pico</song>
+	<song>Philly Nice</song>
+	<song>Blammed</song>
+</week>
+```
+The Week XML above is from Week 3. It does kinda show you how to get going, but it deserves a explaination.
+
+Let's start with the "parent node", it's <syntax lang="xml">&lt;week&gt;</syntax>! That's where the options for the week are defined.
+```xml
+<week name="PICO" chars="pico,bf,gf" sprite="week3">
+```
+
+So far, possible options to set there are:
+- ``name``, which updates the title text for the week (usually on the top right in the Story Mode menu)
+- ``chars``, which controls what characters are shown when selecting the week *(in the order of ``opponent,boyfriend,girlfriend``, no spaces)*.
+- ``sprite``, which tells the engine what image to use for the week button *(has to be at ``menus/storymenu/weeks/``)*. When excluded, the xml filename will be used.
+
+## <h2 id="week-sorting" sidebar="Organizing Weeks">weeks.txt</h2>
+
+There's an optional text file you can use to sort the weeks. It goes under ``./data/weeks/``.
+<br>All you gotta do to order the weeks is just type their file names in the order you'd like:
+```
+week4
+week2
+week6
+```
+This example would order the weeks as Week 4 going first, Week 2 going second and Week 6 being last.
+
+Commentary is possible by typing a pound sign (``#``) before it.
```

Comment:
You can add comments to the list using a pound sign (``#``) at the start of the line.